.github/workflows/tests.yaml

```diff
@@ -37,7 +37,10 @@ jobs:
       pull-requests: write
     strategy:
       matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest]
+        # TODO: Change to macos-latest when Python 3.9 is depcrecated. We're
```

Comment:
s/depcrecated/deprecated/

Reply:
Not keeping this change in this PR. Just added that commit to check whether the macos tests pass under Python 3.9.

Will add this change as part of https://github.com/GoogleCloudPlatform/alloydb-python-connector/issues/388.